tests/casper/order/test_order_estimator.py

```diff
@@ -0,0 +1,68 @@
+"""test order estimator"""
```

Comment:
We should add tests for messages conflicting when we want them to, and not otherwise. Let me know if you want me to add, or if you got it :)

(for all message types)

Reply:
What do you mean? forcing ties and raising an error?